.github/workflows/trigger-builds.yaml

```diff
@@ -0,0 +1,47 @@
+name: Trigger workflow for release
+
+on:
+  workflow_call:
+    inputs:
+      cache-manager-ref:
+        type: string
+        default: main
+      cache-manager-repository:
+        type: string
+        default: gingersnap-project/cache-manager
+      db-syncer-ref:
+        type: string
+        default: main
+      db-syncer-repository:
+        type: string
+        default: pminz/db-syncer
+      operator-ref:
+        type: string
+        default: main
+      operator-repository:
+        type: string
+        default: gingersnap-project/operator
+
+jobs:
+
+  cache-manager-images:
+    uses: gingersnap-project/cache-manager/.github/workflows/create-tag.yml@main
```

Comment:
We discussed the need for the `create-tag.yml` and `publish-release.yaml` files to be combined in order for this job to fail on error, otherwise this job is unaware if a failure has occurred.

The same applies to the db-syncer job.

Reply:
This should be updated to `gingersnap-project/cache-manager/.github/workflows/publish-release.yaml`

Reply:
This should be updated to `gingersnap-project/cache-manager/.github/workflows/publish-release.yaml`. Can we do `@${{ inputs.cache-manager-ref }}` for the `@` version?

Reply:
> Can we do @${{ inputs.cache-manager-ref }} for the @ version?

It seems this isn't possible after all.